packages/paste-website/src/pages/components/file-uploader/index.mdx

```diff
@@ -0,0 +1,495 @@
+| ---------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------- |
+| Invalid file type                                                                              | This file is not an accepted format. You can upload [x], [x] and [x] file formats.                |
+| Exceeds file size limit                                                                        | This file size is too big. You can upload files up to [file limit].                               |
+| Network errors (internet connection dropped while uploading, request timeout, service is down) | Something went wrong with the network. Check your internet connection, then try again.            |
+| Too many uploads in a certain amount of time                                                   | We couldn’t upload so many files so quickly. Try uploading files more slowly, or try again later. |
+| Generic (encountered an internal error)                                                        | Something went wrong. Try uploading your files again.                                             |
+
+Note: be sure to also put the File Uploader in error state and include an error message underneath the FileUploaderDropzone in the case of an upload failure. This will ensure the user notices the failed File Uploader Item, even if it’s at the bottom of a long list of attachments.
+
+### When to use File Uploader
+
+<DoDont>
+  <Do title="Do" body="Use a File Uploader when you need users to upload multiple files." />
+  <Dont
+    title="Don't"
+    body="Use File Uploader for a single file upload if you’re tight on UI space – use File Picker instead."
+  />
+</DoDont>
+<DoDont>
+  <Do title="Do" body="Always include a label with the File Uploader (use the FileUploaderLabel)." />
+  <Dont title="Don't" body="Use FileUploader without FileUploaderLabel." />
+</DoDont>
+<DoDont>
+  <Do
+    title="Do"
+    body="Be sure to communicate to your user when a file is still uploading using the loading state for the FileUploaderItem."
+  />
+  <Dont
+    title="Don't"
+    body="Let users remove uploaded files that may be destructive actions without a warning – instead use the Delete Pattern."
+  />
+</DoDont>
+<DoDont>
+  <Do title="Do" body="Use descriptive help and error text to communicate the types of files that are allowed." />
+  <Dont
+    title="Don't"
+    body="Display a FileUploaderItem
+     in error state without putting the FileUploader itself into error state and providing a helpful error message."
+  />
+</DoDont>
+
+## Usage guide
+
+### API
+
+#### Installation
+
+```bash
+yarn add @twilio-paste/file-uploader - or - yarn add @twilio-paste/core
+```
+
+#### Usage
+
+```jsx
+import {
+  FileUploader,
+  FileUploaderLabel,
+  FileUploaderHelpText,
+  FileUploaderDropzone,
+  FileUploaderDropzoneText,
```

Comment:
Could we include FileUploaderItemList and FileUploaderItem here in case users scroll straight down to the usage section